modin/core/execution/dask/implementations/pandas_on_dask/partitioning/partition.py

```diff
@@ -308,20 +298,16 @@ def ip(self):
         return self._ip_cache
 
 
-def apply_func(partition, func, *args, **kwargs):
+def apply_func(partition, call_queue_entry):
```

Comment:
Let's revert this back. It might be a cause of the slowdown.
```suggestion
def apply_func(partition, func, *args, **kwargs):
```

Reply:
Fixed, and added a note about this in the docstring.